src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/ApplicationClientTests.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Net.Http;
-using Altinn.Platform.Storage.Clients;
+using Altinn.Platform.Storage.Dupicated.Clients;
```

Comment:
Duplicated?

Reply:
yup, they also exist in storage.interface. We have another issue for cleaning up and updating namespaces.